src/components/main-view/main-view.jsx

```diff
@@ -97,10 +92,7 @@ const toggleFav = (id) => {
   } else {
     console.log('Checking if movie is not in favorites');
```

Comment:
`console.logs` are good for debugging, but it's important to remove them from your final code.